cobalt/browser/h5vcc_system/BUILD.gn

```diff
@@ -21,6 +21,7 @@ source_set("h5vcc_system") {
   deps = [
     "//base",
     "//cobalt/browser/h5vcc_system/public/mojom",
+    "//cobalt/configuration",
```

Comment:
@aee-google since this PR is a reland, it'd be great to identify for reviewers the bits that are new. Ideally you'd follow the Chromium ways e.g. https://chromium.googlesource.com/angle/angle/+/main/doc/ContributingCode.md#relanding-a-reverted-cl (that's the angle repo but never mind that).

IIUC the net additions are the Mojo pipe, correct?